### PR TITLE
Adding padding token GPTj config for the tokenizer.

### PR DESCRIPTION
**Pull request by jojivk73**
# What does this PR do?

This PR adds pad_tokens to GPTJ  tokenizer. This was seen as needed when GPTJ uses GPT2 or other tokenizer for GLUE fine-tuning tasks. The changes are in line with ONNX GPTJ Config setting in the same file. If there is an alternate fix that would solve it, that can be added as well.

Fixes # (issue)
Adding padding for GLUE fine-tuning Tasks.

## Who can review?

Models:
GPTJ

Library:
Integrations:

Documentation: @sgugger, @stevhliu and @MKhalusova

HF projects:
- accelerate: [different repo](https://github.com/huggingface/accelerate)
- datasets: [different repo](https://github.com/huggingface/datasets)
- diffusers: [different repo](https://github.com/huggingface/diffusers)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Maintained examples (not research project or legacy):

- Flax: @sanchit-gandhi
- PyTorch: @sgugger
- TensorFlow: @Rocketknight1

 -->

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
No this model was not trained with padding, so this should not be added in the base config.

**Comment by jojivk73** [comment]
@sgugger , The GPTJOnnxConfig uses it though. Is there a better resolution for this in run_glue.py as it requires padding.

**Comment by sgugger** [comment]
The `GPTJOnnxConfig` is not used anymore and only there for backward compatibility. Like I said, you can manually add that `pad_token_id` in the config to suit your needs, but it shouldn't be there by default.
